src/ast.js

```diff
@@ -16,7 +16,9 @@ export class Expression {
   }
 
   toString() {
-    return Unparser.unparse(this);
+    return typeof FEATURE_NO_UNPARSER === 'undefined' ? 
```

Comment:
@jods4 sorry for the delay. This all looks good. If you want to save a few more bits, go ahead and use `return FEATURE_NO_PARSER === undefined ? ... : ...`. We typically don't use the typeof style check for undefined. I think a few contributors have in the past but most of the time we use `something === undefined`. If some jerk redefined what undefined's value is they'll have to deal with it 😉